command.py

```diff
@@ -78,11 +66,13 @@
     'set_returnHomeCoordinates':{'cmd':130, 'type':'ddf8x'}, # lon, lat, alt
     'tare_IMU':                 {'cmd':131, 'type':'fff'},
     'set_IMU':                  {'cmd':132, 'type':'fff'},
-    'set_KDValues':             {'cmd':133, 'type':'fffffff'},
-    'set_KPValues':             {'cmd':134, 'type':'fffffff'},
-    'set_KIValues':             {'cmd':135, 'type':'fffffff'},
+    
     'update_waypoint':          {'cmd':136, 'type':'ddffBxxB'}, # lon, lat, alt, rad, type, id
-    'set_gains':                {'cmd':137, 'type':'hfff'} #first int represents the channel #, 3 floats are KD, KP, KI
```

Comment:
i dont understand. you still have that? the set_roll_rate_gains etc.. commands, only now theyre named logicaly rather than relying on the arbibtrary gain channel mapping on the picpilot